nested-folders/src/Folder.js

```diff
@@ -67,37 +37,26 @@ function makeRandomColor() {
   return '#' + hexColor
 }
 
-function createFolderComponent({id, removable = true}) {
+function createFolderComponent({removable}) {
+  const ChildFolder = createFolderComponent({removable: true})
```

Comment:
Ah woops, thought I tested that! Nice catch 😄 

Fixed by moving the declaration inside of `Folder`.